Mathlib/SetTheory/Ordinal/FixedPointApproximants.lean

```diff
@@ -110,6 +110,41 @@ theorem lfpApprox_add_one (h : x ≤ f x) (a : Ordinal) :
     simp only [Set.mem_setOf_eq]
     use a
 
+theorem lfpApprox_mono₁ (h : f ≤ g) : lfpApprox f ≤ lfpApprox g := by
```

Comment:
Could you state this with `Monotone` too? Perhaps
```suggestion
theorem lfpApprox_mono_left : Monotone (lfpApprox : (α →o α) → _) := by
  intros f g h
```
is enough to make Lean happy.

Reply:
Yes, thanks! That works.